.github/workflows/test_sdxl.yml

```diff
@@ -21,36 +23,70 @@ jobs:
-          # TODO: Let the ref be passed as a parameter to run integration tests.
           path: iree-turbine
+      
+      - name: "Checkout iree"
+        uses: actions/checkout@v4
+        with:
+          repository: iree-org/iree
+          path: iree
 
-      - name: Sync source deps
-        # build IREE from source with -DIREE_BUILD_TRACY=ON if getting tracy profile
+      - name: Python deps
         run: |
+          python3.11 -m venv sdxl_venv
+          source sdxl_venv/bin/activate
           python -m pip install --upgrade pip
-          # Note: We install in three steps in order to satisfy requirements
-          # from non default locations first. Installing the PyTorch CPU
-          # wheels saves multiple minutes and a lot of bandwidth on runner setup.
-          pip install --no-compile --index-url https://download.pytorch.org/whl/cpu \
-            -r ${{ github.workspace }}/iree-turbine//pytorch-cpu-requirements.txt
-          pip install --no-compile --upgrade -r ${{ github.workspace }}/iree-turbine/requirements.txt
-          pip install --no-compile -e ${{ github.workspace }}/iree-turbine/[testing,torch-cpu-nightly]
-          pip install --no-compile --upgrade -r models/requirements.txt
-          pip install --no-compile -e models
+          pip install --no-compile -r ${{ github.workspace }}/iree-turbine/pytorch-cpu-requirements.txt
+          pip install --pre --upgrade -r ${{ github.workspace }}/iree-turbine/requirements.txt
+          pip install --no-compile --pre -e ${{ github.workspace }}/iree-turbine[testing]
+          pip install --no-compile --pre --upgrade -e models -r models/requirements.txt
+          pip uninstall torch torchvision torchaudio -y
+          pip install https://download.pytorch.org/whl/nightly/pytorch_triton_rocm-3.0.0%2B21eae954ef-cp311-cp311-linux_x86_64.whl
+          pip install https://download.pytorch.org/whl/nightly/rocm6.1/torch-2.5.0.dev20240710%2Brocm6.1-cp311-cp311-linux_x86_64.whl
+          pip install https://download.pytorch.org/whl/nightly/rocm6.1/torchvision-0.20.0.dev20240711%2Brocm6.1-cp311-cp311-linux_x86_64.whl
+          pip install https://download.pytorch.org/whl/nightly/rocm6.1/torchaudio-2.4.0.dev20240711%2Brocm6.1-cp311-cp311-linux_x86_64.whl
+          pip uninstall iree-compiler iree-runtime iree-base-compiler iree-base-runtime -y
+      
+      - name: Build IREE
+        run: |
+          python3.11 -m venv iree_venv
+          source iree_venv/bin/activate
+          pip install --upgrade pip
+          pip install -r runtime/bindings/python/iree/runtime/build_requirements.txt
+          cd $GITHUB_WORKSPACE/iree
+          git submodule update --init
+          cmake -S . -B build-release \
+            -G Ninja -DCMAKE_BUILD_TYPE=Release \
+            -DCMAKE_C_COMPILER=`which clang` -DCMAKE_CXX_COMPILER=`which clang++` \
+            -DIREE_HAL_DRIVER_CUDA=OFF \
+            -DIREE_EXTERNAL_HAL_DRIVERS="rocm" \
```

Comment:
`-DIREE_HAL_DRIVER_HIP=ON -DIREE_HAL_DRIVER_HIP=ON` for compiler and runtime backend

Reply:
IREE build no longer needed